test/src/test/kotlin/graphql/nadel/tests/transforms/RemoveFieldTestTransform.kt

```diff
@@ -25,18 +26,18 @@ class RemoveFieldTestTransform : NadelTransform<GraphQLError> {
         service: Service,
         overallField: ExecutableNormalizedField,
     ): GraphQLError? {
-        if (overallField.objectTypeNames
-                .map { executionBlueprint.schema.getType(it) }
-                .any { it == null }
-        ) {
-            return null
-        }
+        val objectType = overallField.objectTypeNames.asSequence()
+            .map {
```

Comment:
could `mapNonNull` be used instead of `map -> filterNotNull`?

Reply:
Eeh yea but it's in test code so eeeeh let's just leave it as is. Too lazy to get it reapproved 🙃